tests/translator/input/simple_function_with_tags_false.yaml

```diff
@@ -0,0 +1,24 @@
+Resources:
+  MyLambdaFunction:
```

Comment:
Can you add some other resources besides SAM function in this tests as well? Just to be thorough

Reply:
Table is here to test another case but I think all the resources refer to the same code when tagging.